Dockerfile

```diff
@@ -1,10 +1,18 @@
-FROM nvcr.io/nvidia/pytorch:22.03-py3
-RUN apt-get update && apt-get install -y --no-install-recommends espeak && rm -rf /var/lib/apt/lists/*
+ARG BASE=nvcr.io/nvidia/pytorch:22.03-py3
+FROM ${BASE}
+RUN apt-get update && apt-get install -y --no-install-recommends python3 python3-pip python3-venv python3-wheel espeak libsndfile1-dev && rm -rf /var/lib/apt/lists/*
+
+# Create and activate virtual env
+ENV VIRTUAL_ENV=/venv
```

Comment:
I think we should use conda to be able to compile MKL to get the best out of CPU runtime. Simple pip install of torch is not as fast and optimized.

Reply:
👍